### ISSUE TITLE
Sys Admin can export XML

### ISSUE THREAD

**Issue by Muraszko** [WGBH/hydradam#37]
We need to be able to export the complete xml that we're ingesting into HydraDAM.

**DONE WHEN**
Sys admin can export the xml files that make up the HydraDAM repository.

**Comment by mccalluc** [comment]
@Muraszko : you can append `.xml` to urls like so: 
http://localhost:3000/files/x059cc103.xml

Is that sufficient?
(I don't trust that the xml is actually making the round trip safely, so I'll check that right now.)

**Comment by Muraszko** [comment]
@mccalluc 
No, that's not good enough.  You can do that as well by clicking the "Export to PBCore" button, but it's one at a time, and a user has to click, save export as...  every time.
 
I'm talking about the raw XML that part of the Fedora data stream, a sys admin being able to export one, or multiple xml files at a time.
